SpatialGDK/Source/SpatialGDKEditorToolbar/Private/SpatialGDKEditorToolbar.cpp

```diff
@@ -339,6 +339,32 @@ bool FSpatialGDKEditorToolbarModule::ValidateGeneratedLaunchConfig() const
 	const USpatialGDKSettings* SpatialGDKRuntimeSettings = GetDefault<USpatialGDKSettings>();
 	const FSpatialLaunchConfigDescription& LaunchConfigDescription = SpatialGDKEditorSettings->LaunchConfigDesc;
 
+	// Ensure that there is an Unreal Server worker in the launch config worker list.
+	if (!LaunchConfigDescription.Workers.ContainsByPredicate([](const FWorkerTypeLaunchSection& Worker) { return Worker.WorkerTypeName == SpatialConstants::ServerWorkerType; }))
+	{
+		const EAppReturnType::Type Result = FMessageDialog::Open(EAppMsgType::YesNo, FText::FromString(FString::Printf(TEXT("The autogenerated launch config does not have a worker section for the UnrealWorker. This is required to launch a deployment. \n\nDo you want to configure your launch config settings now?"))));
```

Comment:
The message here does not say what the cause of the problem is.

"The autogenerated launch config does not have a worker section for the UnrealWorker." - isn't this caused by:

"The launch config description does not have a description for " + {worker type name const goes here} + "."